### ISSUE TITLE
Sagemath uses wrong path for `three.min.js` when calling plot3d

### ISSUE THREAD

**Issue by hqurve** [NixOS/nixpkgs#139478]
### Describe the bug

When using the `three` viewer for plot3d, the generated html file contains the wrong path for `three.min.js`

### Steps To Reproduce

Call the following code in sage

```
x,y=var('x,y')
plot3d(exp(-x^2-y^2), (x,-1,1), (y,-1,1))
```

Then your browser would open the generated file in `~/.sage/tmp/<hostname>/`. 

The browser console would report that 

```
Loading failed for the <script> with source “file:///nix/store/4208lbj85fx0ja3szndn1r6frmbmkq7c-threejs-sage-r122/lib/node_modules/three/r122/three.min.js”

Uncaught ReferenceError: THREE is not defined
```

Upon further inspection, the wanted file is actually at `/nix/store/<hash>/lib/node_modules/three/build/three.min.js` and upon loading the page with this update path, it works.

### Notify maintainers
@timokau @omasanori @7c6f434c @collares 

### Metadata
Please run `nix-shell -p nix-info --run "nix-info -m"` and paste the result.

```console
[user@system:~]$ nix-shell -p nix-info --run "nix-info -m"

 - system: `"x86_64-linux"`
 - host os: `Linux 5.10.68, NixOS, 21.11 (Porcupine)`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.4pre20210908_3c56f62`
 - channels(root): `"nixos-21.11pre314199.f6e9e908ccb"`
 - channels(<username>): `""`
 - nixpkgs: `/nix/var/nix/profiles/per-user/root/channels/nixos`
```

Maintainer information:
```yaml
# a list of nixpkgs attributes affected by the problem
attribute:
# a list of nixos modules affected by the problem
module:
```

**Comment by collares** [comment]
Thanks for the bug report. Unfortunately we can't use vanilla three.js because it does not include some exports used by sage (see the README at https://github.com/sagemath/threejs-sage), but it looks like threejs-sage is indeed installed into the wrong directory. I will fix it today.

**Comment by collares** [comment]
By the way, the threejs-sage installation directory was changed upstream in https://trac.sagemath.org/ticket/30972, which landed in 9.4. I missed this when updating Sage in #130396.
